priam/src/main/java/com/netflix/priam/defaultimpl/CassandraOperations.java

```diff
@@ -126,4 +127,73 @@ public Void retriableCall() throws Exception {
             }
         }.call();
     }
+
+    @Override
+    public List<Map<String, String>> gossipInfo() throws Exception {
```

Comment:
This parsing logic tends to break across the versions, can you add a test case by getting this info from local C* and ensure that parsing logic does not fail in the version that branch is supporting.